journals/apps/journals/models.py

```diff
@@ -353,6 +369,22 @@ class JournalImage(AbstractImage):
         'caption',
     )
 
+    @property
+    def deletion_warning_message(self):
+        """
+        returns the custom deletion warning message if image is being used in journal pages.
+        """
+        warning_message = "Are you sure you want to delete this image?"
+        journal_pages_title = JournalPage.objects.filter(images=self).values_list('title', flat=True)
+        if journal_pages_title:
+            warning_message = "The image '{image_title}' is being used in page(s): {page_titles} " \
+                              "Deleting it will remove the image from the page(s) as well. " \
```

Comment:
use `pluralize` or equivalent for strings, see https://dustindavis.me/django-pluralize-outside-templates/